useScroll/Scrollbar.js

```diff
@@ -60,19 +61,32 @@ const useThemeScrollbar = (props) => {
  * @private
  */
 const ScrollbarBase = memo((props) => {
-	const scrollButtonsRef = useRef();
-
 	const {
 		restProps,
 		scrollbarProps,
 		scrollbarButtonsProps,
 		scrollbarTrackProps
 	} = useThemeScrollbar(props);
 
+	const {
+		prevButtonDisabled,
+		nextButtonDisabled,
+		prevIcon,
+		nextIcon,
+		onClickPrev,
+		onClickNext,
+		prevButtonRef,
+		nextButtonRef,
+		focusOnButton,
+		isOneOfScrollButtonsFocused,
+		updateButtons
+	} = useScrollButtons(scrollbarButtonsProps);
+
+	const {disabled, nextButtonAriaLabel, previousButtonAriaLabel, rtl, vertical} = scrollbarButtonsProps;
```

Comment:
I think `disabled` prop is missing from `scrollbarButtonsProps` returned from `useThemeScrollbar`. Would you please check?

Reply:
Ah, it was not passed to Scrollbar.
I fixed it. https://github.com/enactjs/agate/pull/314/commits/895456565d32cf5328bb526a60d2ce408ede033c

Reply:
I found this modification make a console error because it is crashes with UI code.
I found out that I can't pass the disabled value to scrollbar on the agate side without modifying enact.
```
useScroll.js L1532
	assignProperties('verticalScrollbarProps', {
		disabled: !isVerticalScrollbarVisible,
```

It is designated only as the opposite value of disabledㅣ visible. This is the same behavior as moonstone.
We need to discuss the UX of disabled props.

Reply:
As we discussed in person, 
I reverted that part i set the `disabled` prop on agate usescroll.
https://github.com/enactjs/agate/pull/314/commits/62fb17b5cb82d6fc75e133a37455a31a2215662a